lib/relation-definition.js

```diff
@@ -1369,6 +1411,20 @@ BelongsTo.prototype.related = function (refresh, params) {
   }
 };
 
+
+/**
+ * Define a Promise-based method for the belongsTo relation itself
+ * - order.customer.get(refresh): Load the target model instance asynchronously
+ *
+ * @param refresh
+ * @returns {Promise}
+ */
+BelongsTo.prototype.get = function (refresh) {
+  var cb = utils.createPromiseCallback();
+  return this.related(refresh, cb);
+}
```

Comment:
We should not introduce promise-only APIs, all methods should support both promises and callbacks.

Also the parameter `refresh` is not very intuitive to use, I'd prefer to have two methods instead, e.g. `.get()` and `.getCached()`. For the purpose of this pull request, I am ok with adding the former variant only and wait for user feedback before we add more.

```js
BelongsTo.prototype.get = function (cb) {
  var cb = utils.createPromiseCallback();
  return this.related(true, cb);
};
```

@raymondfeng @ritch thoughts?